static/src/javascripts/projects/common/views/content/block-sharing.html

```diff
@@ -12,3 +12,4 @@
             <%=shareButtons%>
         </div>
 </li>
+=
```

Comment:
Just wanted to check whether this line was supposed to have been added? (I'm not familiar with this templating markup style.)